.vscode/.react/debuggerWorker.js

```diff
@@ -0,0 +1,141 @@
+
```

Comment:
What this be?

Reply:
🤔 🤔 🤔 🤔

Reply:
ah we need this. nuclide uses this to add debugging to react native: https://nuclide.io/docs/platforms/react-native/#debugging

Reply:
not sure if I should ignore it and not push it up?

Reply:
I would ignore it because it's in hidden directories `.vscode/.react`